software/specifically-clementines.yml

```
@@ -9,7 +9,6 @@ platforms:
 tags:
   - Recipe Management
   - Inventory Management
-demo_url: https://www.specificallyclementines.com/
-stargazers_count: 237
+stargazers_count: 238
```

```suggestion
stargazers_count: 237
```

revert unrelated change